package.json

```diff
@@ -70,9 +70,9 @@
     "vue": "^2.6.12"
   },
   "dependencies": {
-    "@storybook/addons": "^6.2.9",
-    "@storybook/client-api": "^6.2.9",
-    "@storybook/vue": "^6.2.9"
+    "@storybook/addons": "^6.3.7",
```

Comment:
These should actually be moved to peerDeps, and support a bigger range like it's done in the react counterpart:

https://github.com/storybookjs/testing-react/blob/main/package.json#L77

Reply:
@yannbf sorry for the late reply. I fixed the dependencies in https://github.com/storybookjs/testing-vue/pull/3/commits/c12a9eaeba09ac8e51ba67b84c9095f0ba6e45f9. 

I also rebased the branch over the latest upstream commit.